src/engine.cc

```diff
@@ -36,7 +36,7 @@
 
 namespace lczero {
 namespace {
-const int kDefaultThreads = 2;
+const int kDefaultThreads = 4;
```

Comment:
I think we should not change default number of threads without testing as a part of larger change.

Reply:
Maybe you have a point. If I revert this back to 2 threads, i would like to include a warning though that CP requires more threads especially on fast GPU systems. Maybe in the option descriptions? But thats not to nice either. Ideas?